vignettes/model-card.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "Model cards for transparent, responsible reporting"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Model Cards for transparent, responsible reporting}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+## What is a "Model Card"?
+
+Good documentation helps us make sense of software, know when and how to use it, and understand its purpose. The same can be true of documentation or reporting for a deployed model, but it can be hard to know where to start. The paper ["Model Cards for Model Reporting" (Mitchell et al. 2019)](https://doi.org/10.1145/3287560.3287596) provides a suggested framework for organizing and presenting the essential facts about a deployed machine learning model. The vetiver package provides an R Markdown template for creating a "Model Card" for a published vetiver model. The template **automates** extracting some information from the model object, and provides **structure** for the model developer where automation is not possible.
+
+Model developers see a nudge to create a model card when they publish a model.
+
+```{r setup}
+library(vetiver)
+library(pins)
+model_board <- board_temp()
+
+cars_lm <- lm(mpg ~ ., data = mtcars)
+v <- vetiver_model(cars_lm, "cars_linear")
+vetiver_pin_write(model_board, v)
+```
+
+(Learn more [about silencing messages like this if desired.](https://rlang.r-lib.org/reference/abort.html#muffling-and-silencing-conditions))
+
+## Accessing the template
+
+To use the vetiver model card template from RStudio, access through `File -> New File -> R Markdown`. This will open the dialog box where you can select from one of the available templates:
+
+![Create a new Model Card](vetiver-templates.png)
+    
+ 
+If you are not using RStudio, you'll also need to install [Pandoc](https://pandoc.org). Then, use the `rmarkdown::draft()` function to create the model card:
+
+```r
+rmarkdown::draft(
+    "my_model_card.Rmd", template = "vetiver_model_card", package = "vetiver"
+)
+```
+
+## Model card outline
+
+There are several basic sections in the model card framework used here.
```

Comment:
Maybe also have a section on who created the model (in demographic terms)?
Such that users know if any minority groups were involved, for example ...

Reply:
I think this would be covered in the **Important aspects/factors** section, if I'm understanding correctly?